Src/WindowsServer/WindowsServer.Shared/AzureWebAppRoleEnvironmentTelemetryInitializer.cs

```diff
@@ -13,13 +13,13 @@
     /// </summary>
     public class AzureWebAppRoleEnvironmentTelemetryInitializer : ITelemetryInitializer
     {
-        /// <summary>Azure Web App name corresponding to the resource name.</summary>
-        private const string WebAppNameEnvironmentVariable = "WEBSITE_SITE_NAME";
-
         /// <summary>Azure Web App Hostname. This will include the deployment slot, but will be same across instances of same slot.</summary>
         private const string WebAppHostNameEnvironmentVariable = "WEBSITE_HOSTNAME";
 
-        private string roleInstanceName;
+        /// <summary>Predefined suffix for Azure Web App Hostname.</summary>
+        private const string WebAppSuffix = ".azurewebsites.net";
```

Comment:
Oh I see below, it's only relevant when this default name is in use, otherwise the full domain name will remain.